src/routes/blog/post/case-study-majik-kids/+page.markdoc

```diff
@@ -0,0 +1,76 @@
+
+# Building like a team of hundreds
+
+In August 2022, Bradley reached out to Phil McCluskey, the eventual Development Lead behind the Majik Kids app, to discuss the product idea. Considering the application idea, he saw Appwrite Cloud as the perfect fit for the backend.
+
+## Why Appwrite was the perfect backend
+
+There were various reasons for him to choose Appwrite: 
+
+* **Improved productivity:** Using Appwrite meant that the engineering team would not have to build their backend or manage their infrastructure, saving a lot of development time.
+* **Support for various languages:** Since Majik Kids is a mobile app, the team chose to build with Flutter to be able to cater to both iOS and Android users. Availability of Appwrite’s Flutter SDK enabled a better developer experience for them.
+* **Reliability:** Phil had been aware of Appwrite in the past and used it in other projects. He had already developed confidence in Appwrite’s ability to support his application functionalities.
+* **Minimal learning curve:** The team discovered that Appwrite fit their software design pattern very well out-of-the-box, and any minor challenges his team faced in working with Appwrite were covered with little effort.
+* **Scalability:** With Appwrite Cloud, one of the biggest worries of the team, scaling their backend infrastructure with growing usage of the application, was instantly diminished as they would no longer have to manage their own infrastructure.
+
+
+## Leveraging different Appwrite products
+
+Keeping Appwrite as their backend, the engineering team started developing the Majik Kids app. Since the application primarily featured audio content in numerous manners, they were required to use a substantial number of binary files, such as MP3 files for the audio, image files for content covers, and PDF files for activity books. **Appwrite Storage** allowed them to quickly host all of these files, set permissions, and leverage the file URLs to consume the content easily.
+
+Phil also developed a sustainable workflow to quickly add any artist’s works (audio story, cover artwork, etc.) to the application, thanks to Appwrite. Since Appwrite offers **SDKs for both client and server-side languages**, including Python, Phil was able to develop a custom Python script that downloads the files from the artist’s shared Google Drive, verifies the integrity of the files, makes any necessary file type conversions, and uses the Appwrite Python SDK to upload all the files to Appwrite Storage.
```

Comment:
Not a review comment, but this is a correct use of etc. by comparison 😆

Reply:
Haha thanks xD